psydac/feec/multipatch/examples_nc/hcurl_eigen_pbms_dg.py

```diff
@@ -0,0 +1,231 @@
+import os
```

Comment:
Please add a short module docstring, which will be visible in the documentation.